docs/source/features/moe.rst

```diff
@@ -4,7 +4,7 @@ Mixture of Experts
 Overview
 --------
 
-NeMo supports Mixture of Experts (MoE) in the transformer layer for NLP models.
+NeMo supports Mixture of Experts (MoE) in the feedforward block of the transformer layer.
```

Comment:
Add Framework.

NeMo Framework supports Mixture of Experts (MoE) in the feedforward block of the transformer layer.